src/service/action-impl.js

```diff
@@ -33,6 +33,10 @@ const ACTION_QUEUE_ = '__AMP_ACTION_QUEUE__';
 /** @const {string} */
 const DEFAULT_METHOD_ = 'activate';
 
+/** @const {!Object<string,!Array<string>>} */
+const ELEMENTS_ACTIONS_MAP_ = {
```

Comment:
This is good for now, but just FYI: it'd be even enough to just check for element name - the most likely error is confusing element's for another. If this set goes out of control - we can simply move the assertion for "unknown action" into the form's action handler itself.